build/parser.js

```diff
@@ -403,13 +403,13 @@ class Parser {
         else if (item.slot === 5) item.category = 'Melee'
         else if (item.slot === 0) item.category = 'Secondary'
         else if (item.slot === 1) item.category = 'Primary'
+        else if (item.uniqueName.includes('/MoaPetHead')) item.category = 'Pets'
         else item.category = 'Misc'
         delete item.isArchwing
         break
 
       case 'Resources':
-        if (item.type === 'Pets') item.category = 'Pets'
```

Comment:
please restore the pets bit, that'll be needed to make sure they stay as part of the pets category, we should re-categorize the `Game/KubrowPet` and `Game/CatbrowPet` as types through `item-types.json` so they correctly get picked up that way. You may need to make sure they're above `Pet` to make sure they trigger first, but i think it's cleaner in case other things get added that should be misc or other categories.

I think we should also add better types for the collars and cosmetics that we can find listed, since those, while not pets, are closer to skins

Reply:
Revising after discussing in dms:

Remove `KubrowPet` from type names and use `PetPowerSuit` instead